Pipfile

```diff
@@ -24,12 +24,13 @@ twilio = "*"
 humanize = "*"
 aiomysql = {editable = true, git = "https://github.com/aio-libs/aiomysql"}
 pyyaml = "*"
+aio_pika = "*"
 
 [dev-packages]
 pytest = "*"
 pytest-mock = "*"
 pytest-cov = "*"
-pytest-asyncio = "*"
+pytest-asyncio = "==0.10.0"
```

Comment:
Reason?

Reply:
The current version breaks some other dependency, this should be temporary until that is resolved.